### ISSUE TITLE
TypeError: find_driver() takes 2 positional arguments but 5 were given

### ISSUE THREAD

**Issue by ralghamdi** [mxrch/GHunt#316]
I was trying to run GHunt from a kali machine with this command "python3 check_and_gen.py" and it shows the below error:

Traceback (most recent call last):
  File "/home/user/ghunt/check_and_gen.py", line 220, in <module>
    driverpath = get_driverpath()
  File "/home/user/ghunt/lib/utils.py", line 214, in get_driverpath
    driverpath_with_version = chromedrivermanager_silent.driver_cache.find_driver(driver.browser_version, driver.get_name(), driver.get_os_type(), driver.get_version())
TypeError: find_driver() takes 2 positional arguments but 5 were given

**Comment by ahmedkhlief** [comment]
me 2

**Comment by najirod** [comment]
Same

**Comment by ItsMalware** [comment]
Same

**Comment by woj-ciech** [comment]
Change line 214 and 215 from lib/utils.py
```python
driverpath_with_version = chromedrivermanager_silent.driver_cache.find_driver(driver.browser_version, driver.get_name(), driver.get_os_type(), driver.get_version())
driverpath_without_version = chromedrivermanager_silent.driver_cache.find_driver("", driver.get_name(), driver.get_os_type(), "")
```
to 
```python
driverpath_with_version = chromedrivermanager_silent.driver_cache.find_driver(driver)
driverpath_without_version = chromedrivermanager_silent.driver_cache.find_driver(driver)
```
and make sure you have chrome browser compatible with the driver.

**Comment by mxrch** [comment]
Fixed in 1e4098aa215761feb08a6132eadb074f4cd823d5 :) you can git pull.
